### PR TITLE
ELY-2166: Added javadoc for org.wildfly.security.sasl.util.SaslMechanismInformation.Names class.

### PR DESCRIPTION
**Pull request by fazer1929**
https://issues.redhat.com/projects/ELY/issues/ELY-2166

### CODE REVIEW

File: sasl/base/src/main/java/org/wildfly/security/sasl/util/SaslMechanismInformation.java [hunk 1]
```diff
@@ -55,6 +55,9 @@
  */
 public final class SaslMechanismInformation {
 
+    /**
+     *  A Collection of Sasl Mechanism names.
```
Comment (skyllarr) [inline sasl/base/src/main/java/org/wildfly/security/sasl/util/SaslMechanismInformation.java]: @fazer1929 Thank you for the PR. Just a minor nitpick from me, the word collection in java makes it seem like it is a [Collection](https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html), so I would rephrase it

Comment (fazer1929) [reply sasl/base/src/main/java/org/wildfly/security/sasl/util/SaslMechanismInformation.java]: Got it, will change it

Comment (skyllarr) [reply sasl/base/src/main/java/org/wildfly/security/sasl/util/SaslMechanismInformation.java]: @fazer1929 Thanks! There is also a class [Group](https://docs.oracle.com/javase/7/docs/api/java/security/acl/Group.html) in java. So maybe something like `The class providing Sasl Mechanism Names.`  is sufficient.

Comment (fazer1929) [reply sasl/base/src/main/java/org/wildfly/security/sasl/util/SaslMechanismInformation.java]: sorry for that 😅. I have updated the PR

**Comment by skyllarr** [comment]
@fazer1929 Thank you

**Review by skyllarr — APPROVED** [review]

**Comment by fjuma** [comment]
Thanks @fazer1929!
